src/libcore/sync/atomic.rs

```diff
@@ -18,11 +18,11 @@
 //!
 //! Each method takes an [`Ordering`] which represents the strength of
 //! the memory barrier for that operation. These orderings are the
-//! same as [LLVM atomic orderings][1]. For more information see the [nomicon][2].
+//! same as the [C++ atomic orderings][1]. For more information see the [nomicon][2].
```

Comment:
not C11?

Reply:
AFAIK C just copies C++ here. And there were some fixes since C++11.

But indeed the Nomicon says "C11"...

Reply:
Yea; I mostly want to have a consistent saying one way or the other. (I don't have a preference and you are the field expert.)

Reply:
(Adding Cc in an edit does not trigger email notifications.)

The [C page on `memory_order`](https://en.cppreference.com/w/c/atomic/memory_order) is much less informative, so I'd prefer to link to the C++ page. And that page currently reflects C++20. I am not sure if it is worth trying to pin a page from some older C++ version; AFAIK all the changes have been bugfixes that we certainly want.

I assume @Gankra wrote "C11" as most people just say "C11 concurrency". It's shorter to write than "C/C++ from 2011 and later".

Cc @nikomatsakis @Gankra

Reply:
I could replace "C++" by "C/C++" everywhere, and we could do the same in the Nomicon?

Reply:
Maybe say C++20 to be more precise if we can all agree?

Reply:
Sure, whatever. We just better make sure we bump everything consistently when a new version comes out...

Reply:
Yep; want to change to the new wording? r=me with that (also cc @rust-lang/lang)

Reply:
Done, also see https://github.com/rust-lang-nursery/nomicon/pull/168.

Reply:
This is a minor thing, but I'm glad to see this fixed, I've been correcting Gankro on this since at least [three years ago](https://www.reddit.com/r/rust/comments/43vjk4/what_are_rusts_memory_ordering_guarantees/czm7u36/).

(I suspect those people using "C11 concurrency" are exactly those who read it from Nomicon, because I very rarely see it used outside of Rust community.)

Reply:
@petrochenkov so do you think we should further clarify [the comment I added on this in the Nomicon](https://github.com/rust-lang-nursery/nomicon/pull/168/files#diff-7b3a0d960bfa7f3e187d7dab979a641cR9)?

Reply:
It's not just Gankra though; here's a [talk by some of my colleagues](https://people.mpi-sws.org/~viktor/wmc/c11.pdf) also using "C11". I think it is generally understood as a short-hand for "C/C++11" -- but it is indeed somewhat confusing.

Reply:
I don't know, looks ok as is.